app/src/main/java/phuchh/com/music_60/data/model/Track.java

```diff
@@ -0,0 +1,133 @@
+package phuchh.com.music_60.data.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class Track implements Parcelable {
+
+    private String mArtworkUrl;
+    private boolean mDownloadable;
```

Comment:
```suggestion
    private boolean mIsDownloadable;
```em sửa lại tên cho hợp lý